src/commons/report.py

```diff
@@ -32,18 +33,16 @@
 LOGGER = logging.getLogger(ROOT)
 
 
-# pylint: disable-msg=too-many-branches
-def log_status(parsed_input: dict, corio_start_time: datetime, test_failed: str, **kwargs):
+def log_status(parsed_input: dict, corio_start_time: datetime, **kwargs):
     """
     Log execution status into log file.
 
     :param parsed_input: Dict for all the input yaml files.
     :param corio_start_time: Start time for main process.
-    :param test_failed: Reason for failure is any.
-    # :param dt_string: datetime string used in report.
+    # :param test_failed: Reason for failure is any.
     # :param terminated_tests: terminated tests from workload.
     """
-    terminated_tests = kwargs.get("terminated_tests", None)
+    test_failed = kwargs.get("test_failed")
```

Comment:
What is the difference between terminated tests and test_failed.
Please add appropriate description.

Reply:
Both parameters being used to mark test status in reports, added description for them.